api/src/main/java/org/arquillian/smart/testing/strategies/affected/Tests.java

```diff
@@ -0,0 +1,42 @@
+package org.arquillian.smart.testing.strategies.affected;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation used to set which production classes are tested in current test.
+ * By default it appends all classes defined in all attributes.
+ *
+ * If none of the attributes are set, then all production classes with same package as test and its subpackages are added.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+@Repeatable(TestsList.class)
+@Documented
+public @interface Tests {
```

Comment:
By looking at this annotation my first thought was that it will execute list of test classes defined in it, so the name is a bit ambiguous. Maybe `@Covers` or `@ComponentUnderTest` makes it more self-explanatory?